src/main/java/com/sevenmap/logger.java

```diff
@@ -0,0 +1,25 @@
+import java.io.IOException;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+public class logger {
+    public static void main(String[] args) {
+        Logger logger = Logger.getLogger("login");
+        FileHandler fileHandler;
+        try {
+          fileHandler = new FileHandler("C:\\logfile.log", true);
+          logger.addHandler(fileHandler);
+          logger.setLevel(Level.ALL);
+          SimpleFormatter formatter = new SimpleFormatter();
+          fileHandler.setFormatter(formatter);
+          logger.log(Level.WARNING, "Warning message");
+          logger.log(Level.SEVERE, "Severe message");
+          logger.log(Level.INFO, "Info message from Throwable object", new Throwable("Throwable exception"));
```

Comment:
I plainly don't understand what you meant to do here, why do you generate these test logs inside the main function ? Why doesn't the class have a `log(message)` method of its own ? The logger class is supposed to be an alternative to the classic `System.out.println()` call, and in this case, I don't see how anyone could use it that way.